tests/src/rules/no-named-default.js

```diff
@@ -0,0 +1,31 @@
+import { test, SYNTAX_CASES } from '../utils'
+import { RuleTester } from 'eslint'
+
+const ruleTester = new RuleTester()
+    , rule = require('rules/no-named-default')
+
+ruleTester.run('no-named-default', rule, {
+  valid: [
+    test({code: 'import bar from "./bar";'}),
+    test({code: 'import bar, { foo } from "./bar";'}),
+
+    // es7
+    test({ code: 'import bar from "./bar";', parser: 'babel-eslint' }),
+    test({ code: 'import bar, { foo } from "./bar";', parser: 'babel-eslint' }),
```

Comment:
I'd say it's safe yes. At this point and AFAIK, the only difference is that one parser might consider some syntax invalid while another accepts it (and it goes both ways now with the latest version of Espree).